var/spack/repos/builtin/packages/blitz/package.py

```diff
@@ -17,6 +17,15 @@ class Blitz(AutotoolsPackage):
     version("1.0.2", sha256="500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2")
 
     depends_on("python@3:", type="build")
+    depends_on("m4", type="build")
+    depends_on("autoconf", type="build")
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
+
+    patch("fujitsu_compiler_specfic_header.patch", when="%fj")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@alalazo 
I have added a comennt. This patch fixes makefile and include header file to build with Fujitsu compiler.